cmd/monaco/download/download_configs.go

```diff
@@ -69,7 +70,12 @@ func (d DefaultCommand) DownloadConfigsBasedOnManifest(fs afero.Fs, cmdOptions m
 		return fmt.Errorf("environment %q was not available in manifest %q", cmdOptions.specificEnvironmentName, cmdOptions.manifestFile)
 	}
 
-	printUploadToSameEnvironmentWarning(env.Url.Value, env.Auth.Token.Value)
+	err := cmdutils.VerifyClusterGen(manifest.Environments{env.Name: env})
```

Comment:
Why does this happen both in the `_command` and here? Could we move it to one place? 
I assume the call here is because we haven't loaded the manifest yet before - so maybe the version check for `direct` can also move in here?

Reply:
your comment doesn't make sense to me because _[...]move in here[...]_ refers to a place inside `DownloadConfigsBasedOnManifest`
atm this check is only performed when dealing with manifests.

Reply:
With "in here" i meant from the `download_command` to the `download_configs` file (which IIRC also contains the functions for direct download?)
Currently the (simpler) version check happens as part of cobra validation in one case, and in or logic in the _cofings.go file in the other - which is strange to me (but no huge issue)

Fine for now, but let's revisit when we've decided how we should deal with oAuth for direct downloads.